util/misc.go

```diff
@@ -430,7 +434,26 @@ type SequenceTable interface {
 // LoadTLSCertificates loads CA/KEY/CERT for special paths.
 func LoadTLSCertificates(ca, key, cert string) (tlsConfig *tls.Config, err error) {
 	if len(cert) == 0 || len(key) == 0 {
-		return
+		tempStoragePath := config.GetGlobalConfig().TempStoragePath
+		cert = tempStoragePath + "/cert.pem"
```

Comment:
We did not have a windows version, though. May we use `filepath.Join`, which is portable across all platforms?

Reply:
+1, @dveeden Could you update this?

Reply:
:white_check_mark:  https://github.com/pingcap/tidb/pull/24141/commits/4dcf272c05e1164cef0100aad2fe033c119d78cd